kernel/kernel-config-4.9.x

```diff
@@ -1241,9 +1241,85 @@ CONFIG_HAVE_IDE=y
 #
 CONFIG_SCSI_MOD=y
 # CONFIG_RAID_ATTRS is not set
-# CONFIG_SCSI is not set
-# CONFIG_SCSI_DMA is not set
+CONFIG_SCSI=y
+CONFIG_SCSI_DMA=y
```

Comment:
Can I ask - do we know if we need the SCSI_DMA set?
I was going to ask the same of the SG, but the virtio scsi driver does have SG code, and I can see how that could be used. Having DMA for a purely virtual device feels slightly odd though (but maybe it too is tied into how SG is passed/used?).

Reply:
@grahamwhaley This is enabled by default with virtio-scsi, maybe pulled in with SG as well.